src/modules/launcher/Plugins/Microsoft.Plugin.Uri/UriHelper/ExtendedUriParser.cs

```diff
@@ -12,12 +12,12 @@ namespace Microsoft.Plugin.Uri.UriHelper
     public class ExtendedUriParser : IUriParser
     {
         // When updating this method, also update the local method IsUri() in Community.PowerToys.Run.Plugin.WebSearch.Main.Query
-        public bool TryParse(string input, out System.Uri result, out bool isWebUri)
+        public bool TryParse(string input, out System.Uri webUri, out System.Uri systemUri)
         {
             if (string.IsNullOrEmpty(input))
             {
-                result = default;
-                isWebUri = false;
+                webUri = default;
+                systemUri = null;
```

Comment:
if we set the default values (default and null) globally before parsing input we can have less code and don't have to set those generic values within conditions. What do you think?

And why don't we set both to null or default?

Reply:
Sounds good to me, but I'll fix it later when I have a proper keyboard, current away